spyder/plugins/editor/extensions/snippets.py

```diff
@@ -700,17 +700,9 @@ def cursor_changed(self, line, col):
             self.inserting_snippet = False
             return
 
-        ignore_calls = {'undo', 'redo'}
         node, nearest_snippet, _ = self._find_node_by_position(line, col)
         if node is None:
-            stack = inspect.stack()
```

Comment:
Why is this code removed @impact27

Reply:
@andfoy some context would help :-)

Reply:
The code here want to know if it is called from "undo" or "redo". It does that by inspecting the call stack, which is not a very nice way to do this as it requires loading the source files for the entire call stack! I therefore changed that to add a `self.editor.is_undoing` to check for undo and redos instead

Reply:
Understood!